### PR TITLE
Legacy features

### PR DESCRIPTION
**Pull request by oxyc**
- Adds a button style to TinyMCE editors. Useful if there are any ACF WYSWIYG Fields in a block for example.
- Adds the style button to tinymce editors and remove foreground color.
- Normalize youtube/vimeo oembed urls so they always work
- Enforce some default oembed url parameters for youtube/vimeo, eg show related within channel, modest branding.
- Provide a `App\build_url()` utility function for rebuilding an url as provided by `parse_url()`

### CODE REVIEW

**Comment by toffebjorkskog** [comment]
Nice

File: app/filters.php [hunk 1]
```diff
@@ -36,3 +36,55 @@
...
+/**
+ * Modify oEmbed URL parameters.
+ */
+add_filter('embed_oembed_html', function ($cache, $url, $attr, $post_id) {
+    preg_match('/src="([^"]*)"/i', $cache, $sources);
+    if (!empty($sources)) {
+        $src = $sources[1];
+    }
+    if (!empty($src) && !empty($url)) {
+        $is_youtube = strpos($src, 'youtube') !== false;
```
Comment (silentnoodlemaster) [inline app/filters.php:49]: is it always youtube when embedding? they have short url service youtu.be

Comment (oxyc) [reply app/filters.php:49]: Good point but I tested it and verified that the oEmbed does resolve to the full URL before this check comes. So it works.

Comment (silentnoodlemaster) [reply app/filters.php:49]: Great!

**Pull request merged**